pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/AdminOperationService.java

```diff
@@ -56,6 +58,9 @@ public String startOperation(String operationTypeId, Map<String, String> attribu
             case MEDICAL_RECORD_INIT:
                 adminOperation = new MedicalRecordInitService();
                 break;
+            case LEGACY_KIT_RESAMPLE:
```

Comment:
Does this operation only apply to "legacy" kits?  What is a legacy kit?  Alternatively, we can name the type something like `UPDATE_COLLAB_IDS` and avoid the nuance.